clang/test/SemaSYCL/device_global.cpp

```diff
@@ -0,0 +1,61 @@
+// RUN: %clang_cc1 -fsycl-is-device -std=c++17 -sycl-std=2020 -ast-dump -verify %s | FileCheck %s
```

Comment:
Is it ok to simply remove AST checking since the attributes added only use simple handling?

Reply:
> Is it ok to simply remove AST checking since the attributes added only use simple handling?

We should have at least one, so if something inadvertently changes in the future, we can catch it.

Reply:
We should probably also add basic attribute diagnostics. I see we missed it for https://github.com/intel/llvm/pull/5576. By basic checks I mean attribute applied on wrong subject/incorrect arguments, etc. I guess in this case and #5576, we don't have arguments, so I guess only subject check is required. Can you please add this this PR or your follow-up diagnostics PR?

Reply:
https://github.com/intel/llvm/pull/5597/commits/1c53934b473317419bf4f2e73eceffff985ff0e6 - how about this?